apps/web/package.json

```diff
@@ -76,6 +76,7 @@
     "handlebars": "^4.7.7",
     "ical.js": "^1.4.0",
     "ics": "^2.37.0",
+    "jose": "^4.13.1",
```

Comment:
I could be mistaken but didnt we already have a package to sign JWTs?

Reply:
I introduced "jose" in the past and Alex found a way to not depend on it.